tests/test_model/test_detr.py

```diff
@@ -0,0 +1,47 @@
+# import colossalai
+# import pytest
+# import torch
+
+# from titans.model.detr import DETR
+# from colossalai.global_variables import tensor_parallel_env as tp_env
+# from colossalai.testing import rerun_if_address_is_in_use
+# from tests.utils import run_with_parallel_config
+
+# BATCH_SIZE = 4
+# IMAGE_SIZE = 224
+# PATCH_SIZE = 16
+# NUM_HEADS = 4
+# IN_CHANS = 3
+# HIDDEN_SIZE = 32
+
+
+# def run_detr(data, img_size, patch_size, in_chans, hidden_size, num_heads):
```

Comment:
You can just remove this file, if everything get commented.

Reply:
fixed